pom.xml

```diff
@@ -76,13 +76,10 @@
 
   <repositories>
     <repository>
-      <id>all-snapshots</id>
-      <name>Liveramp Repositories</name>
-      <url>http://repository.liveramp.com/artifactory/libs-snapshot-local</url>
```

Comment:
Do users of this package need to update anything?

Reply:
This change only updates where the build process for captain downloads its dependencies from. It shouldn't have any impact on users 🤞

Reply:
Oh, right. This is to tell it where to download from. My bad. Noob mistake